model-optimizer/mo/graph/graph.py

```diff
@@ -147,7 +147,7 @@ def in_ports(self, control_flow=False):
         for idx in self._in_ports:
             if control_flow or 'control_flow' not in self._in_ports[idx] or not self._in_ports[idx]['control_flow']:
                 ports.update({idx: self.in_port(idx, control_flow=control_flow)})
-        return dict_to_ordered_dict(ports, func=lambda t: str(t))
+        return dict_to_ordered_dict(ports, func=lambda t: int(t))
```

Comment:
This fix is incorrect because control flow ports have indices: "control_flow_<port_id>".

Reply:
Updated, now we use `str()` when `control_flow=True` and `int()` otherwise.